chart/values.yaml

```diff
@@ -39,11 +39,30 @@ gitvote:
   addr: 0.0.0.0:9000
 
   # GitHub configuration
+  #
+  # The app require the following permissions:
+  # - Repository permissions:
+  #   - Checks: read/write
+  #   - Contents: read
+  #   - Issues: read/write
+  #   - Metadata: read
+  #   - Pull requests: read/write
+  # - Organization
+  #   - Members: read
+  #
+  # Subscribe to events:
+  # - Issue Comment
+  #
+  # Activate webhook with URL: https://example.com/api/events
```

Comment:
Maybe we could add here that the GitVote service expects events to be posted at the `/api/events` endpoint, to make it clearer that it's the path that must be used when setting up the webhook URL in the GitHub app.